src/_includes/templates/forms/accessibility-feedback-form.html

```diff
@@ -11,183 +13,46 @@
-                Telephone number*
-            </label>
-            <input class="input__long"
-                   id="accessibility-form_telephone-number"
-                   type="tel"
-                   required>
-        </div>
+        {{ form.input('Zip code', 'text', '1-3', true) }}
 
-        <div class="form-l_col form-l_col-1-2">
-            <label class="form-label-header"
-                   for="accessibility-form_fax-number">
-                Fax number
-            </label>
-            <input class="input__long"
-                   id="accessibility-form_fax-number"
-                   type="tel">
-        </div>
+        {{ form.input('Telephone number', 'tel', '1-2', true) }}
 
-        <div class="form-l_col form-l_col-1-2">
-            <label class="form-label-header"
-                   for="accessibility-form_email-address">
-                Email address
-            </label>
-            <input class="input__long"
-                   id="accessibility-form_email-address"
-                   type="email">
-        </div>
+        {{ form.input('Fax number', 'tel', '1-2', false) }}
 
-        <div class="form-l_col form-l_col-1-2">
-            <label class="form-label-header"
-                   for="accessibility-form_contact-time">
-                Best time to reach you
-            </label>
-            <div class="custom-select">
-                <select class="custom-select_select"
-                        data-placeholder="Select Time"
-                        id="accessibility-form_contact-time">
-                      <option value="">Select time</option>
-                      {# TODO: Add additional times. #}
-                      <option>8:00 a.m</option>
-                 </select>
-                 <span class="custom-select_icon
-                              cf-icon
-                              cf-icon-down"></span>
-            </div>
-        </div>
+        {{ form.input('Email address', 'email', '1-2', false) }}
 
-        <div class="form-l_col form-l_col-1">
-            <label class="form-label-header"
-                   for="accessibility-form_complaint-desc">
-                Nature of the accessibility problem*
-            </label>
-            <textarea class="input__long"
-                      id="accessibility-form_complaint-desc"
-                      required>
-            </textarea>
-        </div>
+        {# TODO: Add additional times to form select below. #}
 
-        <div class="form-l_col form-l_col-1">
-            <label class="form-label-header"
-                   for="accessibility-form_address-request-material">
-                Web address (URL) of requested material
-            </label>
-            <input class="input__long"
-                   id="accessibility-form_address-request-material"
-                   type="url">
-        </div>
+        {{ form.select('Best time to reach you',
+                       'Select time',
+                       ['8:00 a.m'],
```

Comment:
I would wait until we know what times are going in before we design to heavily here. They might decide to go with time blocks like "Morning, Afternoon, Evening" or have only a few times available.